README.md

```diff
@@ -35,38 +35,50 @@ We'll implement the tool in several incremental phases to get some basic functio
  
 Based on Tim's feedback, instead of using `<xsdl:include/>` as originally proposed, change it so that it can be used to either include _all_ the scripts in the referenced file:
 
-	<Binding>
-		<Definition>
-			<Script>
-				<Transforms>
-					<xsdl:EspRequest>
-						<xsdl:include-from file="service-scripts.xml" all="true"/>
-						<!-- 
-								All the transforms defined in service-scripts.xml
-								are included here in this element
-						-->
-					</xsdl:EspRequest>
-				</Transforms>
-			</Script>
-		</Definition>	
-	</Binding>
+  <Binding>
+    <Definition>
+      <Script>
+        <Transforms>
+          <EspRequest>
+            <xsdl:include file="service-scripts.xml"/>
+		<!-- 
+			By default all the transforms defined in service-scripts.xml
+			are included here in this element
+		-->
+          </EspRequest>
+        </Transforms>
+      </Script>
+    </Definition>	
+  </Binding>
 
 or just include some by name:
 
-	<Binding>
-		<Definition>
-			<Script>
-				<Transforms>
-					<xsdl:EspRequest>
-						<xsdl:include-from file="service-scripts.xml" all="false">
-							<xsdl:include transform="GenericOutOfBand"/>
-							<xsdl:include transform="UserOutOfBand"/>
-						</xsdl:include-from>
-					</xsdl:EspRequest>
-				</Transforms>
-			</Script>
-		</Definition>	
-	</Binding>
+  <Binding>
+    <Definition>
+      <Script>
+        <Transforms>
+          <EspRequest>
+            <xsdl:include file="service-scripts.xml" transform="GenericOutOfBand"/>
+            <xsdl:include file="service-scripts.xml" transform="UserOutOfBand"/>
+          </EspRequest>
+        </Transforms>
+      </Script>
+    </Definition>	
+  </Binding>
+
+or as a list of names:
```

Comment:
Are you saying to allow both styles or we need to choose one or the other? Even though it is more terse I always feel like putting delimiters in attribute values is a little like misuse of XML. If you guys are OK with it though and prefer it I'll go along.

Reply:
Let's stick with the simplest form for now.  We can optimize how it gets processed internally.